src/index.ts

```diff
@@ -0,0 +1,231 @@
+import * as util from 'util';
```

Comment:
Rename the directory from `src` to `source`. @sindresorhus and I agreed upon this name in another project.